sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TablesExtensionConfigProvider.cs

```diff
@@ -70,6 +71,7 @@ public void Initialize(ExtensionConfigContext context)
 
             binding.BindToCollector<TableEntity>(CreateTableWriter);
 
+            binding.BindToInput<ParameterBindingData>(CreateParameterBindingData);
```

Comment:
Is the ordering significant here?  I'm curious if adding this may change the existing behavior by preferring `ParameterBindingData` for scenarios that currently use the attribute binding provider.

Reply:
I think it might be, I know that when I implemented this for Blob the order did matter. 

@aishwaryabh we should make sure to test all existing scenarios E2E to make sure we don't break existing scenarios (inproc and out of proc)

Reply:
Yes the order did matter here. If I placed the new line after the `TableAttributeBindingProvider` the parameter data binding was not able to bind to input properly as it kept erroring out on `ParameterDataBinding` not having a default constructor. I can validate that this doesn't break in-proc and out of proc scenarios

Reply:
Validated that in-proc functions also work with these changes

Reply:
Changed the ordering so that we call `TableAttributeBindingProvider` first, and then call `ParameterBindingData` input binding calls